### PR TITLE
Added status-bar color-animation during the transition QuizActivity -> CategorySelectionActivity

### PR DESCRIPTION
**Pull request by shchurov**
When you press back on QuizActivity, status bar changes its color only in the end of animation.

### CODE REVIEW

**Comment by shchurov** [comment]
Changed commit message to a more apporpriate one

File: app/src/main/java/com/google/samples/apps/topeka/helper/TransitionHelper.java [hunk 1]
```diff
@@ -23,16 +23,21 @@
...
      */
     public static Pair[] createSafeTransitionParticipants(@NonNull Activity activity,
-            @Nullable Pair... otherParticipants) {
+            boolean excludeStatusBar, @Nullable Pair... otherParticipants) {
         // Avoid system UI glitches as described here:
         // https://plus.google.com/+AlexLockwood/posts/RPtwZ5nNebb
         View decor = activity.getWindow().getDecorView();
-        View statusBar = decor.findViewById(android.R.id.statusBarBackground);
+        View statusBar = null;
+        if (!excludeStatusBar) {
```
Comment (keyboardsurfer) [inline app/src/main/java/com/google/samples/apps/topeka/helper/TransitionHelper.java]: Please invert this to `includeStatusBar` and document it within the method doc.

File: app/src/main/java/com/google/samples/apps/topeka/helper/TransitionHelper.java [hunk 2]
```diff
@@ -23,16 +23,21 @@
...
+            statusBar = decor.findViewById(android.R.id.statusBarBackground);
+        }
         View navBar = decor.findViewById(android.R.id.navigationBarBackground);
 
         // Create pair of transition participants.
         List<Pair> participants = new ArrayList<>(3);
-        addNonNullViewToTransitionParticipants(statusBar, participants);
+        if (!excludeStatusBar) {
```
Comment (keyboardsurfer) [inline app/src/main/java/com/google/samples/apps/topeka/helper/TransitionHelper.java]: This check is redundant as the method call to `addNonNullViewToTransitionParticipants` already checks for null values.

**Comment by shchurov** [comment]
updated

**Comment by keyboardsurfer** [comment]
Released with 14a2fe222e104d96b115e90b42f48748c56166f5
I've re-written the commit message once more in the process of rebasing this.

**Pull request closed without merging**